
Prevent the -ldflags output from including -L/usr/lib, since this will break
e.g. the Subversion build will then pick up system libraries rather than 
local ones.

--- SWIG-1.3.21/configure.in.ldflags	2004-09-30 15:26:03.000000000 +0100
+++ SWIG-1.3.21/configure.in	2004-09-30 15:28:49.000000000 +0100
@@ -16,16 +16,16 @@
 #define SWIG_LANG               "-tcl"
 
 /* Values returned by SWIG when invoked with the -ldflags option */
-#define SWIG_GUILE_RUNTIME      "-L" LIBDIR " -lswigguile" RELEASE_SUFFIX
-#define SWIG_GUILESCM_RUNTIME   "-L" LIBDIR " -lswigguilescm" RELEASE_SUFFIX
-#define SWIG_MZSCHEME_RUNTIME   "-L" LIBDIR " -lswigmz" RELEASE_SUFFIX
-#define SWIG_PERL_RUNTIME       "-L" LIBDIR " -lswigpl" RELEASE_SUFFIX
-#define SWIG_PIKE_RUNTIME       "-L" LIBDIR " -lswigpike" RELEASE_SUFFIX
-#define SWIG_PHP_RUNTIME        "-L" LIBDIR " -lswigphp4" RELEASE_SUFFIX
-#define SWIG_PYTHON_RUNTIME     "-L" LIBDIR " -lswigpy" RELEASE_SUFFIX
-#define SWIG_RUBY_RUNTIME       "-L" LIBDIR " -lswigrb" RELEASE_SUFFIX
-#define SWIG_TCL_RUNTIME        "-L" LIBDIR " -lswigtcl" RELEASE_SUFFIX
-#define SWIG_CHICKEN_RUNTIME    "-L" LIBDIR " -lswigchicken" RELEASE_SUFFIX
+#define SWIG_GUILE_RUNTIME       "-lswigguile" RELEASE_SUFFIX
+#define SWIG_GUILESCM_RUNTIME    "-lswigguilescm" RELEASE_SUFFIX
+#define SWIG_MZSCHEME_RUNTIME    "-lswigmz" RELEASE_SUFFIX
+#define SWIG_PERL_RUNTIME        "-lswigpl" RELEASE_SUFFIX
+#define SWIG_PIKE_RUNTIME        "-lswigpike" RELEASE_SUFFIX
+#define SWIG_PHP_RUNTIME         "-lswigphp4" RELEASE_SUFFIX
+#define SWIG_PYTHON_RUNTIME      "-lswigpy" RELEASE_SUFFIX
+#define SWIG_RUBY_RUNTIME        "-lswigrb" RELEASE_SUFFIX
+#define SWIG_TCL_RUNTIME         "-lswigtcl" RELEASE_SUFFIX
+#define SWIG_CHICKEN_RUNTIME     "-lswigchicken" RELEASE_SUFFIX
 ])
 
 dnl Checks for programs.
